README.md

```diff
@@ -106,10 +106,12 @@ To build and run tests, use:
 go test ./...
```

Comment:
we probably need the `export GO111MODULE=on` on line 95

Reply:
these commands appear to work just fine with `GO111MODULE=auto`, is there a benefit to be had for us/the developer by changing that to `on`?

Reply:
Are you sure? I'm pretty sure that `auto` is the default, and that `GO111MODULE=auto` will produce broken builds if building within `go/src/` and `go version < 1.13`